app/src/main/java/mozilla/lockbox/view/ItemDetailFragment.kt

```diff
@@ -183,14 +180,14 @@ class ItemDetailFragment : BackableFragment(), ItemDetailView {
 
     // used for feature flag
     override fun showKebabMenu() {
-        toolbar.kebabMenu.visibility = View.VISIBLE
-        toolbar.kebabMenu.isClickable = true
+        toolbar.kebabMenuButton.visibility = View.VISIBLE
+        toolbar.kebabMenuButton.isClickable = true
```

Comment:
Is `FeatureFlag.CRUD_EDIT` and `FeatureFlag.CRUD_DELETE` relevant here? i.e. will we ever ship one without the other?